pandas/parser.pyx

```diff
@@ -42,6 +42,9 @@ import sys
 
 cdef bint PY3 = (sys.version_info[0] >= 3)
 
+cdef int QUOTE_NONE = 3  # csv.QUOTE_NONE
+cdef int QUOTE_NONNUMERIC = 2  # csv.QUOTE_NONNUMERIC
```

Comment:
However, in light of your point about the namespacing, `io.common` makes sense.  I'll just import them from `csv` into the namespace.